client/modules/Preview/EmbedFrame.jsx

```diff
@@ -227,7 +227,7 @@ function injectLocalFiles(files, htmlFile, options) {
   const accessiblelib = sketchDoc.createElement('script');
   accessiblelib.setAttribute(
     'src',
-    'https://cdn.jsdelivr.net/gh/processing/p5.accessibility@v0.1.1/dist/p5.accessibility.js'
+    'https://cdn.jsdelivr.net/gh/processing/p5.accessibility@0.1.1/dist/p5-accessibility.js'
```

Comment:
I'm looking at the [options for formulating jsdelivr URLs](https://www.jsdelivr.com/).  We can use `@0` to get the latest version matching `0.x`.  We can also use `.min.js` to create a minified version, but in that case it actually uses the `p5.` version??? 🤷 https://cdn.jsdelivr.net/gh/processing/p5.accessibility@0/dist/p5.accessibility.min.js

But the URL that you have here definitely fixes the "not found" issue ✔️